data/preset/escola.yaml

```diff
@@ -2,11 +2,13 @@
 vars:
   app-banner:
     shortdef: 'Banner.Dynamic(mode: text, expand: true)'
-  app-brand:
+  app-banner-text:
     type: ContentGroup.StaticText
     properties:
       halign: center
       label: ''
+  app-brand:
```

Comment:
Are you sure this `app-brand` should be named the same as the other `app-brand`s? This is overlaid in the corner like a watermark, and would be omitted on most uses of the template except for FNDE, and the other ones are a more integral part of the design.

Reply:
> would be omitted on most uses of the template except for FNDE

In general, variables are optional to use (if a variable exist, it doesn't mean the user will be forced to use the var). In particular, if no watermark is provided, the behavior of this default (Banner.App) is to hide it. Taking these two things into account, I don't see a problem if this gets ignored by whoever else uses this preset in the future.

Also, the `app-brand` var is (as of this PR) used in `video_list`, `library-list`and `escola` presets. (trying to think as someone using this) If I see this watermark in one preset and can be customized with a `app-brand` var then, I would expect that if I see this same watermark in another preset, then I would also expect a similar var in that preset.

I think consistency is a good reason for keeping the name.

Reply:
But in `video_list` it's not a watermark. The default value there is equal to `*app-banner`, so it's not consistent. (I guess I should have commented on the `video_list` one, instead of this one.)

Reply:
I still believe we shouldn't focus too much on what the current default value is, I am not suggesting we should encourage them to put a ContentGroup, but say someone wants to put a StaticText with "www.mycompany.com", it would be a valid override in all of these 3 places, because conceptually that is what that place is for (regardless of whether one uses a themeableImage or a dynamiclogo as default).